src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Machine - truck started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Machine - truck stopped working");
```

Comment:
Similarly, the message should clearly indicate that it is a 'Truck' that has stopped working. Consider changing 'Machine - truck' to 'Truck' to make the message more informative.